lib/utilities/monitoringHandler.js

```diff
@@ -64,6 +64,36 @@ const httpResponseSizeBytes = new client.Summary({
     help: 'Cloudserver HTTP response sizes in bytes',
 });
 
+const quotaEvaluationDuration = new client.Histogram({
+    name: 's3_cloudserver_quota_evaluation_duration_seconds',
+    help: 'Duration of the quota evaluation operation',
+    labelNames: ['action', 'code', 'type'],
```

Comment:
we may have either account or bucket quotas, but could also have both : how is this managed in the `type` label? should we maybe have 2 labels instead (`bucket` and `account`) ?

Reply:
Right now we have 3 different strings (bucket, account, bucketAccount), if we want to add more types later, I thought it made sense to have a single label for it...